index.html

```diff
@@ -0,0 +1,39 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+<section class="wrap">  
+    
+
+<header class="headr">
+
+    <div class="flex">
+        <a href="#home" class="logo">My Logo</a>
+        <img src="unt/Union.png" alt="" class="bars">
```

Comment:
- [ ] I would kindly suggest you wrap your `hamburger` icon inside a `nav` tag, The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. 👍 👍